common-utils/src/common/deployment/__init__.py

```diff
@@ -294,22 +272,26 @@ def _get_schedule_arg(self):
             str: Command line argument to use for setting the deployment's schedule.
         """
         schedule = self.schedule
-        if isinstance(schedule, CronSchedule):
-            return f"--cron '{schedule.cron}'"
-        elif isinstance(schedule, RRuleSchedule):
-            return f"--rrule '{schedule.rrule}'"
-        elif isinstance(schedule, IntervalSchedule):
-            return f"--interval '{schedule.interval.seconds}'"
-        else:
+        # we should only schedule for main flows
+        if not DEPLOYMENT_TYPE == "main":
             return ""
+        else:
+            if isinstance(schedule, CronSchedule):
+                return f"--cron '{schedule.cron}'"
+            elif isinstance(schedule, RRuleSchedule):
+                return f"--rrule '{schedule.rrule}'"
+            elif isinstance(schedule, IntervalSchedule):
+                return f"--interval '{schedule.interval.seconds}'"
+            else:
+                return ""
 
     def push_deployment(
         self,
-        storage_path: str,
+        project_name: str,
         infrastructure: str,
         flow_path: Path,
         flow_function_name: str,
-        skip_upload: bool,
+        flow_name: str,
```

Comment:
we also need to provide the period delimited flow_name, which is covered further down the module review.